lscolors.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/nsf/termbox-go"
+)
+
+func init() {
+	if lscolors := os.Getenv("LSCOLORS"); lscolors != "" {
```

Comment:
It seems that `LS_COLORS` is much more capable than `LSCOLORS`. It makes sense that if both of these variables are set, then `LS_COLORS` should override `LSCOLORS`. This way BSD/OSX folks would have a way to use this extra capability. A minor nitpick in this code, if both variables are set at the same time, then I think they are both parsed. If you move `LS_COLORS` block above the other and add an early return, it may look more intentional and it would only be parsed once when both variables are set.

Reply:
Well, `init()` first parses `LSCOLORS` and then overwrites the result with `LS_COLORS`. At first I thought that it makes sense (something like "augment `LSCOLORS` with `LS_COLORS`") but now I'm starting to have my doubts.